### PR TITLE
- Update packages & Explorer bug fix

### PR DESCRIPTION
**Pull request by tgiphil**

### CODE REVIEW

File: Source/Mosa.Tool.Explorer/MainForm.cs [hunk 1]
```diff
@@ -467,6 +467,11 @@ private void CompileAll()
 		if (Compiler == null)
 			return;
 
+		if (MosaSettings.EmitBinary & !Directory.Exists(MosaSettings.DefaultFolder))
```
Comment (AnErrupTion) [inline Source/Mosa.Tool.Explorer/MainForm.cs:470]: Seems like you forgot an ampersand here.

**Review by charsleysa — APPROVED** [review]
LGTM

File: Source/Mosa.Tool.Explorer/MainForm.cs (hunk 1 above)
Comment (tgiphil) [reply Source/Mosa.Tool.Explorer/MainForm.cs]: Nice catch!

**Review by AnErrupTion — APPROVED** [review]
LGTM

**Pull request merged**